bchrpc/bchrpc.proto

```diff
@@ -734,7 +736,8 @@ message Transaction {
         bytes previous_script = 6;
         // The bitcoin addresses associated with this input.
         string address = 7;
-        SlpToken slp_token = 8;
+        CashToken cash_token = 8;
```

Comment:
Same here, CashToken data should come after SLP, that way the ordering of existing fields doesn't change.